pkg/config/config.go

```diff
@@ -38,6 +38,10 @@ const (
 	HTTPPortInternal = uint32(8081)
 	// HTTPSPortExternal is the port for external HTTPS availability.
 	HTTPSPortExternal = uint32(8443)
+	// HTTPPortProbProxyProtocol is the port for prob when proxy-protocol is enabled
```

Comment:
If we use `8090` and `9443` for probe by default (i.e. net-kourier always use `8090` and `9443` ports to probe regardless proxy-protocol enabled or disabled), the code especially `lister.go`  becomes much simpler?
Do you think it is a bad idea?

Reply:
I think it's a good idea, I'll do that